### PR TITLE
Add Reputation KB entry

### PR DESCRIPTION
**Pull request by ErebusZ**

### CODE REVIEW

**Review by PiranhaSa — APPROVED** [review]

File: MOBILE_CLIENT/COMMON/_INFO/REPUTATION_SCAN/context.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+The 'Domain Name and IP Address Reputation Report' assesses the reputation of domain names and IP addresses associated with your application. This report collects data from various APIs and public databases to determine if a domain name or IP address has been identified as potentially malicious.
```
Comment (elyousfi5) [inline MOBILE_CLIENT/COMMON/_INFO/REPUTATION_SCAN/context.md:1]:
```suggestion
The 'Domain Name and IP Address Reputation Report' assesses the reputation of the domain names and IP addresses associated with your application. This report collects data from various APIs and public databases to determine if a domain name or IP address has been identified as potentially malicious.
```

File: MOBILE_CLIENT/COMMON/_INFO/REPUTATION_SCAN/context.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+Low reputation IPs and domain names may not have proper security measures in place, making your application vulnerable to security breaches and data leaks. These entities could be susceptible to hacking, phishing, and other cyberattacks, putting your users' data and privacy at risk.
+
+Low reputation IPs and domain names may not be reliable in terms of uptime and availability. They could experience frequent downtime, slow response times, or even sudden shutdowns, causing disruptions to your application's functionality and user experience.
+
+Some low reputation domains and IPs may infringe upon intellectual property rights, contain malicious content, or engage in unethical or illegal activities. Associating your application with such entities can lead to legal issues, reputational damage, and potential regulatory compliance problems.
+
+If your application relies on low reputation domain names, it can negatively affect your search engine rankings. Search engines like Google consider the reputation and quality of the domains you link to. If you associate with low-quality domains, your application's visibility in search results may suffer.
```
Comment (3asm) [inline MOBILE_CLIENT/COMMON/_INFO/REPUTATION_SCAN/context.md:9]: Focus should be on the malicious aspect of the IP/domain which could indicate a compromise. We don't need a context for this, just put it in the description and remove context.md.

File: MOBILE_CLIENT/COMMON/_INFO/REPUTATION_SCAN/meta.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "risk_rating": "info",
```
Comment (3asm) [inline MOBILE_CLIENT/COMMON/_INFO/REPUTATION_SCAN/meta.json:2]:
```suggestion
    "risk_rating": "high",
```

File: MOBILE_CLIENT/COMMON/_INFO/REPUTATION_SCAN/meta.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    "short_description": "Report for domain name and IP addresses reputation score using multiple scanning APIs and databases.",
```
Comment (3asm) [inline MOBILE_CLIENT/COMMON/_INFO/REPUTATION_SCAN/meta.json:3]: Does the risk vary depending on if the domain is malicious or not?

**Review by 3asm — CHANGES_REQUESTED** [review]

File: MOBILE_CLIENT/COMMON/_INFO/REPUTATION_SCAN/meta.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{
+    "risk_rating": "high",
```
Comment (PiranhaSa) [inline MOBILE_CLIENT/COMMON/_INFO/REPUTATION_SCAN/meta.json:2]: Why is risk rating high and the Reputation_scan is in the _INFO folder ?

Comment (ErebusZ) [reply MOBILE_CLIENT/COMMON/_INFO/REPUTATION_SCAN/meta.json:2]: thats a new change

**Review by 3asm — APPROVED** [review]

**Pull request merged**